### DISCUSSION TITLE
(Date)Time Range Picker

### DISCUSSION THREAD

**Discussion by hilkeheremans** [evidence-dev/evidence#2332] [Ideas]
## Feature Description

A date/time range picker instead of just the date range picker we have now.

(I see how you implemented the current date picker and realize this is not trivial, but I wanted to add this in here in case you were considering switching to a different one at one point, so that you may included it in the 'requirements' if deemed useful)

## Goal of Feature

Make it easier for our users to select specific time ranges.

## Current Solution / Workarounds

Use custom Svelte components to obtain the functionality I need. That works, it's just cumbersome.

## Alternatives

'Drag to zoom' selection on the charts themselves can provide a partial workaround, but that is also not a support feature atm :-)

**Comment by mcrascal** [comment]
Would you expect this as an option in the DateRange picker, or as a distinct component?
